robotpy_ext/control/toggle.py

```diff
@@ -65,14 +66,15 @@ def get(self):
             else:
                 return False
 
-    def __init__(self, joystick, button, debouncePeriod=None):
+    def __init__(self, joystick: wpilib.Joystick, button: int, debounce_period=None):
         """
-        :param joystick: wpilib.Joystick that contains the button to toggle
-        :param button: Value of button that will act as toggle. Same value used in getRawButton()
+        :param joystick: :class:`wpilib.Joystick` that contains the button to toggle
+        :param button: Number of button that will act as toggle. Same value used in `getRawButton()`
+        :param debounce_period: Period to wait before registering a new button press.
```

Comment:
I don't know what units `debounce_period` should be in from just looking at this.